custom_components/tuya_local/devices/wilfa_haze_hu400bc_humidifier.yaml

```diff
@@ -58,14 +58,16 @@ secondary_entities:
         name: speed
         mapping:
           - dps_val: level_0
-            value: 20
+            value: 0

```

Comment:
A speed of zero results in the fan being turned off by the switch.  I'm not sure that is correct behaviour for a setting of level_0.